diff --git a/tensorflow/core/kernels/roi_align_op.cu.cc b/tensorflow/core/kernels/roi_align_op.cu.cc
index 886f4bc81f..0a801dceb8 100644
--- a/tensorflow/core/kernels/roi_align_op.cu.cc
+++ b/tensorflow/core/kernels/roi_align_op.cu.cc
@@ -1298,8 +1298,8 @@ __global__ void WriteUprightBoxesOutput(const CudaLaunchConfig nboxes,
     d_image_out_rois[base_idx + 0] = image_index;
     d_image_out_rois[base_idx + 1] = box.x;
     d_image_out_rois[base_idx + 2] = box.y;
-    d_image_out_rois[base_idx + 3] = box.z;
-    d_image_out_rois[base_idx + 4] = box.w;
+    d_image_out_rois[base_idx + 3] = box.z + 1.0f;
+    d_image_out_rois[base_idx + 4] = box.w + 1.0f;
   }
 }
 
@@ -2395,4 +2395,4 @@ REGISTER_KERNEL_BUILDER(
     tensorflow::sami::BoxIntersectionOverUnion);
 
 }  // namespace tensorflow
-#endif
\ No newline at end of file
+#endif
