diff --git a/tensorflow/python/keras/backend.py b/tensorflow/python/keras/backend.py
index fc2e8aa..e3081c0 100644
--- a/tensorflow/python/keras/backend.py
+++ b/tensorflow/python/keras/backend.py
@@ -594,7 +594,8 @@ def _has_nchw_support():
       bool: if the current scope device placement would support nchw
   """
   explicitly_on_cpu = _is_current_explicit_device('CPU')
-  gpus_available = bool(_get_available_gpus())
+  #gpus_available = bool(_get_available_gpus())
+  gpus_available = True
   return not explicitly_on_cpu and gpus_available